tools/ansible/tasks/wsu/main.yaml

```diff
@@ -199,3 +204,26 @@
       fail:
         msg: "CNI Configuration error"
       when: '"CNI configuration completed successfully" not in bootstrap_out.stderr'
+
+    - name: Copy kube-proxy to C:\k
+      win_copy:
+        src: "{{ win_temp_dir.path }}\\kube-proxy.exe"
+        dest: "C:\\k\\kube-proxy.exe"
+        remote_src: yes
+
+    - name: Copy required powershell script to the temporary directory
+      win_copy:
+        src: "powershell/hns.psm1"
+        dest: "{{ win_temp_dir.path }}\\hns.psm1"
+
+    - name: Get source-vip
+      win_shell: 'Import-Module {{ win_temp_dir.path }}\\hns.psm1; $net = (Get-HnsNetwork | where { $_.Name -eq "OpenShiftNetwork" }); $endpoint = New-HnsEndpoint -NetworkId $net.ID -Name VIPEndpoint; Attach-HNSHostEndpoint -EndpointID $endpoint.ID -CompartmentID 1; (Get-NetIPConfiguration -AllCompartments -All -Detailed | where { $_.NetAdapter.LinkLayerAddress -eq $endpoint.MacAddress }).IPV4Address.IPAddress.Trim()'
+      register: source_vip
+
+    - name: Create kube-proxy Windows Service
+      win_service:
```

Comment:
If the node is rebooted, is the service configured to start? If not, please ensure that it does.